projects/gloo/api/v1/enterprise/options/ai/ai.proto

```diff
@@ -9,714 +9,738 @@ option (extproto.equal_all) = true;
-    ```
-    defaults:
-    - field: "temperature"
-      value: 0.5
-    - field: "max_tokens"
-      value: 100
-    ```
-  */
+  // Provide defaults to merge with user input fields.
+  // Defaults do _not_ override the user input fields, unless you explicitly set `override` to `true`.
   repeated FieldDefault defaults = 5;
 
+  // The type of route to the LLM provider API.
   enum RouteType {
+    // The default route type for prompt and response LLM APIs.
     CHAT = 0;
+    // Stream responses to a client, which allows partial results for certain prompts.
     CHAT_STREAMING = 1;
   }
 
-  // The type of route this is, currently only CHAT and CHAT_STREAMING are supported
+  // The type of route to the LLM provider API. Currently, `CHAT` and `CHAT_STREAMING` are supported.
   RouteType route_type = 6;
 }
 
+// Provide defaults to merge with user input fields.
+// Defaults do _not_ override the user input fields, unless you explicitly set `override` to `true`.
+//
+// Example overriding the system field for Anthropic:
+// ```yaml
+// # Anthropic doesn't support a system chat type
+// defaults:
+// - field: "system"
+//   value: "answer all questions in french"
+// ```
+//
+// Example setting the temperature and overriding `max_tokens`:
+// ```yaml
+// defaults:
+// - field: "temperature"
+//   value: 0.5
+// - field: "max_tokens"
+//   value: 100
+// ```
 message FieldDefault {
-  // Field name
+  // The name of the field.
   string field = 1;
-  // Field Value, this can be any valid JSON value
+  // The field default value, in JSON format.
   google.protobuf.Value value = 2;
-  // Whether or not to override the field if it already exists
+  // Whether to override the field's value if it already exists.
+  // Defaults do _not_ override the user input fields by default,
+  // unless you explicitly set this field to `true`.
   bool override = 3;
 }
 
-
+// Configuration settings for a Postgres datastore.
 message Postgres {
-  // Connection string to the Postgres database
+  // Connection string to the Postgres database. For example, to use a vector database
+  // deployed to your cluster, your connection string might look similar to
+  // `postgresql+psycopg://gloo:gloo@vector-db.default.svc.cluster.local:5432/gloo`.
   string connection_string = 1;
-  // Name of the table to use
+  // Name of the collection table to use.
   string collection_name = 2;
 }
 
-
-
+// Configuration for the backend LLM provider authentication token.
 message Embedding {
 
-  // OpenAI embedding
+  // Embedding settings for the OpenAI provider.
   message OpenAI {
     oneof auth_token_source {
+      // The authorization token that the AI gateway uses to access the OpenAI API.
+      // This token is automatically sent in the `Authorization` header of the
+      // request and prefixed with `Bearer`.
       SingleAuthToken auth_token = 1;
       // re-use the token from the backend
       // google.protobuf.Empty inherit_backend_token = 3;
     }
```

Comment:
No, this is embedding, currently the model is always the same. It may become useful if we expose `custom_host` here so that the user can use different endpoints or models which support the `OpenAI` API